src/__tests__/insert.test.ts

```diff
@@ -51,7 +51,7 @@ describe.only("CRUD operations work", () => {
 
     const savedCollection = await collectionRepository.save(testCollection)
     const readCollection = await collectionRepository.findOne({
-      _id: savedCollection._id,
+      _id: savedCollection.id,
```

Comment:
It appears that the `collectionRepository.save(testCollection)` call populates the `id` property, but not `_id`. I haven't tracked down where the change is in typeorm that made this difference.

This is the only place in the current kaws codebase that needed this update, because other places that reference `_id` have just _read_ from the db, which appears to populate `_id` in all cases.